### PR TITLE
Create new list and add the movie to this list.

### PR DESCRIPTION
**Pull request by pink-bunny**

### CODE REVIEW

File: src/components/CreateNewListModal/component.js [hunk 1]
```diff
@@ -4,21 +4,30 @@ import { Field } from 'formik';
...
-  errors
+  errors,
+  triggerComponent: TriggerComponent,
+  triggerProps
 }) => (
   <>
-    <Icon onClick={showModal} type="plus-circle" />
+    <TriggerComponent
+      {...triggerProps} /* eslint-disable-line react/jsx-props-no-spreading */
+      onClick={() => {
```
Comment (plzen) [inline src/components/CreateNewListModal/component.js]: Don't use inline functions in components (UI part), move this to container

File: src/pages/Movie/TopInfo/AddToListPopover/component.js [hunk 2]
```diff
@@ -1,36 +1,29 @@
...
-                id={item.id}
-                name={item.name}
-              />
-            </div>
-          ))}
-        </>
+        <PopoverContent
+          list={list}
+          addToList={addToList}
+          closePopover={() => handleVisiblePopover(false)}
```
Comment (plzen) [inline src/pages/Movie/TopInfo/AddToListPopover/component.js]: Move inline function to container

File: src/components/CreateNewListModal/component.js (hunk 1 above)
Comment (pink-bunny) [reply src/components/CreateNewListModal/component.js]: Done

File: src/pages/Movie/TopInfo/AddToListPopover/component.js (hunk 2 above)
Comment (pink-bunny) [reply src/pages/Movie/TopInfo/AddToListPopover/component.js]: Done

**Pull request merged**